languages/shop-standards-de_DE.po

```diff
@@ -148,31 +161,40 @@ msgstr ""
 msgid "Hide sale percentage bubble"
 msgstr "Sale: Prozentangabe verbergen"
 
-#: src/WooCommerce.php:180 src/WooCommerce.php:349
+#: src/WooCommerce.php:180 src/WooCommerce.php:360
 msgid "Hide add to cart button"
 msgstr "Warenkorb-Button verstecken"
 
-#: src/WooCommerce.php:187 src/WooCommerce.php:367
+#: src/WooCommerce.php:187 src/WooCommerce.php:378
 msgid "Price comparison focus product"
 msgstr "Preisbeobachtung Fokus-Produkt"
 
 #: src/WooCommerce.php:204
 msgid "Internal product notes"
 msgstr "Interne Anmerkungen"
 
-#: src/WooCommerce.php:332
+#: src/WooCommerce.php:235 src/WooCommerce.php:413
+msgid ""
+"The entered GTIN isn't unique and couldn't be saved. Please enter a unique "
+"GTIN and save again. All other changes have been saved."
+msgstr ""
+"Die eingegebene GTIN wird bereits verwendet und konnte nicht gespeichert "
+"werden. Sie muss geändert und erneut gespeichert werden. Alle anderen "
+"Änderungen wurden bereits gespeichert."
+
+#: src/WooCommerce.php:343
 msgid "GTIN:"
 msgstr ""
 
-#: src/WooCommerce.php:341
+#: src/WooCommerce.php:352
 msgid "ERP/Inventory ID:"
 msgstr "ERP/Inventar ID:"
 
-#: src/WooCommerce.php:357
+#: src/WooCommerce.php:368
 msgid "Variation has insufficient images"
-msgstr "Variation hat ein unzureichendes Bilder"
+msgstr "Variation hat unzureichende Bilder"
```

Comment:
Is this changes intended? To me, seems not related with the current feature. 🤔

Reply:
Not related, but wrong anyway.